main.js

```diff
@@ -0,0 +1,119 @@
+//Towers of Hanoi Eval
+console.log("Please refer to instructions above");
+console.log("Starter code arrMove(0, 0)");
+console.log("Cheat code winningMoves()");
+//Gameboards
+
+var board = [
+  [5, 4, 3, 2, 1], 
+  new Array(), 
+  new Array()
+];
+
+var board2 = [
+  [5, 4, 3, 2, 1], 
+  new Array(), 
+  new Array()
+];
+
+
+//Cheat Code
+var winningMoves =  function () {
+  arrMove(0, 2)
+  arrMove(0, 1);
+  arrMove(2, 1)
+  arrMove(0, 2)
+  arrMove(1, 0)
+  arrMove(1, 2)
+  arrMove(0, 2)
+  arrMove(0, 1);
+  arrMove(2, 0)
+  arrMove(0, 1);
+  arrMove(2, 0)
+  arrMove(1, 0)
+  arrMove(2, 1)
+  arrMove(0, 2)
+  arrMove(0, 1);
+  arrMove(2, 0)
+  arrMove(0, 1);
+  arrMove(0, 2)
+  arrMove(1, 2)
+  arrMove(1, 0)
+  arrMove(2, 0)
+  arrMove(1, 2)
+  arrMove(0, 1)
+  arrMove(0, 2)
+  arrMove(1, 2)
+  arrMove(2, 0)
+  arrMove(2, 1)
+  arrMove(0, 2)
+  arrMove(2, 1)
+  arrMove(2, 0)
+  arrMove(1, 2)
+  arrMove(2, 0)
+  arrMove(0, 2)
+  arrMove(1, 0)
+  arrMove(2, 0)
+  arrMove(1, 2)
+  arrMove(0, 2)
+  arrMove(0, 1);
+  arrMove(2, 1)
+  arrMove(0, 2)
+  arrMove(1, 0)
+  arrMove(1, 2)
+  arrMove(0, 2)
+  };
+  
+//Displays Original Board
+console.table(board);
+
+
+//Function to map board play by play, move pieces, throw error if a larger number is placed on top of another and reset board
+
+function arrMove(oldIndex, newIndex) {
+  board = board.map( function (someVar){
```

Comment:
`someVar` is not very semantic, i.e. it doesn't give a description of what the thing is. I would call this `peg`.